{"name": "User login with incorrect credentials", "status": "passed", "attachments": [{"name": "stdout", "source": "fd02e7a2-de3e-47b0-ba31-b596b6b063c6-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "payload", "value": "{'email': 'yozhik_ya@yandex.ru_new', 'password': 'yozhik-123'}"}], "start": 1732029579249, "stop": 1732029579555, "uuid": "8fccb509-451a-447e-9178-fc25589f64bd", "historyId": "03c5f6126ebf4e6d899b5044cf2a413b", "testCaseId": "76148a48b7585b4a8f8a20e34c40f04e", "fullName": "tests.user.test_user_login.TestUserLogin#test_user_login_incorrect_credentials", "labels": [{"name": "suite", "value": "User Tests"}, {"name": "parentSuite", "value": "API Tests"}, {"name": "subSuite", "value": "User Login Tests"}, {"name": "host", "value": "ubuntu22"}, {"name": "thread", "value": "10772-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.user.test_user_login"}]}